index.js

```diff
@@ -163,6 +165,30 @@ function createMainWindow() {
 }
 
 app.once("browser-window-created", (event, win) => {
+	// User agents are from https://developers.whatismybrowser.com/useragents/explore/
+	const originalUserAgent = win.webContents.userAgent;
+	const userAgents = {
+		mac: "Mozilla/5.0 (Macintosh; Intel Mac OS X 12.1; rv:95.0) Gecko/20100101 Firefox/95.0",
```

Comment:
Login seems to work fine on Mac with the fix and this user-agent: `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.93 Safari/537.36`

Reply:
Would be willing to test a version with the electron upgrade reverted on Windows. As far as I understood that would also revert the remote package setup, right? So would like to re-check it once.

Reply:
@th-ch You can't really test the fix unless you are shadowbanned 😅